docs/source/user-guide/concepts/installing-with-conda.rst

```diff
@@ -0,0 +1,35 @@
+=====================
+Installing with conda
+=====================
+
+.. image:: /img/installing-with-conda.png
+    :align: right
+
+.. _installing-with-conda:
+
+
+To install conda packages, in the terminal or an Anaconda Prompt, run::
+
+  conda install [packagename]
+
+
+During the install process, files are extracted into the
+``install`` prefix, except for files in the ``info/``
+directory. Installing the files of a conda package into an
+environment can be thought of as changing the directory to an
+environment, and then downloading and extracting the .zip file
```

Comment:
"downloading and extracting the .zip" 
I think should can be more general. Conda packages now are .tar.bz2's, the new package format will be .conda. So maybe this can be "downloading and extracting the artifact"